controllers/imageclusterinstall_controller.go

```diff
@@ -325,8 +365,8 @@ func (r *ClusterConfigReconciler) removeBMHImage(ctx context.Context, bmhRef *re
 	return nil
 }
 
-func (r *ClusterConfigReconciler) configDirs(config *relocationv1alpha1.ClusterConfig) (string, string, error) {
-	lockDir := filepath.Join(r.Options.DataDir, "namespaces", config.Namespace, config.Name)
+func (r *ImageClusterInstallReconciler) configDirs(clusterInstall *relocationv1alpha1.ImageClusterInstall) (string, string, error) {
```

Comment:
In some functions, it's `ici` in others it's `clusterInstall`.
Try sticking with one of them (I prefer `ibi`)